docs/development-environment.md

```diff
@@ -40,25 +40,25 @@ In order to test the provider, you can run `make test`.
 
 *Note:* Make sure no `AWS_ACCESS_KEY_ID` or `AWS_SECRET_ACCESS_KEY` variables are set, and there's no `[default]` section in the AWS credentials file `~/.aws/credentials`.
 
-```sh
+```console
 $ make test
 ```
 
 In order to run the full suite of Acceptance tests, run `make testacc`.
```

Comment:
given the size of the repo, maybe not realistic any more to suggest running the full suite of tests 😅 or you could make a note that there will be a lot of tests running? the command usage below is a helpful starting point tho to then link out to the other guide for more detailed usage

Reply:
i peeped the azure provider and i like how the noted the testing there as an example: https://github.com/hashicorp/terraform-provider-azurerm/blob/main/DEVELOPER.md#developing-the-provider for an example